src/utils/daos_hdlr.c

```diff
@@ -2244,83 +2064,45 @@ file_lstat(struct cmd_args_s *ap, struct file_dfs *file_dfs,
 		/* POSIX returns -1 on error and sets errno
 		 * to the error code
 		 */
-		if (rc != 0) {
+		if (rc != 0)
 			rc = errno;
-		}
 	} else if (file_dfs->type == DAOS) {
-		rc = stat_dfs(ap, file_dfs, path, buf);
+		rc = dfs_sys_stat(file_dfs->dfs_sys, path, 0, buf);
 	} else {
-		fprintf(ap->errstream,
-			"File type not known, file=%s, type=%d\n",
-			path, file_dfs->type);
+		DH_PERROR_SYS(ap, rc, "File type not known '%s' type=%d", path, file_dfs->type);
```

Comment:
rc would be zero here, but should probably be EINVAL

Reply:
@daltonbohning thanks, I guess I missed a couple of these.